### ISSUE TITLE
`with_processes=True` will not provide child payload data for keyed histograms whose keys are not present in parent

### ISSUE THREAD

**Issue by chutten** [mozilla/python_moztelemetry#50]
```python
subset = get_pings_properties(pings, ["payload/keyedHistograms/BLOCKED_ON_PLUGIN_INSTANCE_INIT_MS"], with_processes=True)
```

`subset` will not have `_parent` and `_children` variants for pings with e10s enabled because `BLOCKED_ON_PLUGIN_INSTANCE_INIT_MS` only ever reports in the child process for e10s. 

This is why: https://github.com/mozilla/python_moztelemetry/blob/b41cee792107e84cd42be3b40485e03538345ac8/moztelemetry/spark.py#L465

It will only fetch histograms for keys that are present in the parent (ie, the `kh_keys`)

...so, maybe with `with_processes=True` we should merge into `kh_keys` all of the parent's and children's keys?

**Comment by chutten** [comment]
A workaround is to specify the key you want. Then it will correctly call `_get_merged_histograms` for you. Unfortunately, you then run afoul of #51

**Comment by vitillo** [comment]
Fixed in #55 .
